test/built-ins/Proxy/enumerate/return-trap-result-done.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.11
+description: >
+    Trap returns a done iterable result.
```

Comment:
This description could use some work. Instead of "Trap returns a done iterable result", what about: "Trap returns an iterator whose IteratorResult does not contain a `value` property". Likewise, the file name could be `return-trap-result-no-value.js`